spec/src/main/asciidoc/chapters/revision.asciidoc

```diff
@@ -2,6 +2,17 @@
 [[revision_history]]
 Revision History
 ----------------
+3.0
+[[version_three_zero]]
+~~~
+
+Deprecate requirement for Facelets support
```

Comment:
One question. It states "Deprecate requirement for Facelets", but we actually removed Faclets completely from the spec. Is this the correct way to go? Should it be "Removed requirement for Faclets" instead?

Reply:
You're right, "Removed requirement for Facelets" is far better. I'll open a separate PR for this. Because there was no API to deprecate, I decided to remove the requirement completely as those view engines are more or less a implementation detail.